app/models/miq_server/worker_management/monitor/class_names.rb

```diff
@@ -1,149 +1,10 @@
-    ManageIQ::Providers::Amazon::StorageManager::S3::RefreshWorker
-    ManageIQ::Providers::Google::CloudManager::RefreshWorker
-    ManageIQ::Providers::Google::NetworkManager::RefreshWorker
-    ManageIQ::Providers::AnsibleTower::AutomationManager::RefreshWorker
-    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::RefreshWorker
-    ManageIQ::Providers::Foreman::ConfigurationManager::RefreshWorker
-    ManageIQ::Providers::Foreman::ProvisioningManager::RefreshWorker
-    ManageIQ::Providers::Hawkular::MiddlewareManager::RefreshWorker
-    ManageIQ::Providers::Hawkular::DatawarehouseManager::RefreshWorker
-    ManageIQ::Providers::Kubernetes::ContainerManager::RefreshWorker
-    ManageIQ::Providers::Lenovo::PhysicalInfraManager::RefreshWorker
-    ManageIQ::Providers::Openshift::ContainerManager::RefreshWorker
-    ManageIQ::Providers::Microsoft::InfraManager::RefreshWorker
-    ManageIQ::Providers::Redhat::InfraManager::RefreshWorker
-    ManageIQ::Providers::Openstack::CloudManager::RefreshWorker
-    ManageIQ::Providers::Openstack::NetworkManager::RefreshWorker
-    ManageIQ::Providers::Openstack::InfraManager::RefreshWorker
-    ManageIQ::Providers::StorageManager::CinderManager::RefreshWorker
-    ManageIQ::Providers::StorageManager::SwiftManager::RefreshWorker
-    ManageIQ::Providers::Vmware::CloudManager::RefreshWorker
-    ManageIQ::Providers::Vmware::NetworkManager::RefreshWorker
-    ManageIQ::Providers::Vmware::InfraManager::RefreshWorker
-    ManageIQ::Providers::Nuage::NetworkManager::RefreshWorker
-    MiqScheduleWorker
-    MiqPriorityWorker
-    MiqWebServiceWorker
-    MiqEmsRefreshCoreWorker
-    MiqVimBrokerWorker
-    ManageIQ::Providers::Vmware::CloudManager::EventCatcher
-    ManageIQ::Providers::Vmware::InfraManager::EventCatcher
-    ManageIQ::Providers::Redhat::InfraManager::EventCatcher
-    ManageIQ::Providers::Openstack::CloudManager::EventCatcher
-    ManageIQ::Providers::Openstack::NetworkManager::EventCatcher
-    ManageIQ::Providers::Openstack::InfraManager::EventCatcher
-    ManageIQ::Providers::StorageManager::CinderManager::EventCatcher
-    ManageIQ::Providers::Amazon::CloudManager::EventCatcher
-    ManageIQ::Providers::Azure::CloudManager::EventCatcher
-    ManageIQ::Providers::AnsibleTower::AutomationManager::EventCatcher
-    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::EventCatcher
-    ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher
-    ManageIQ::Providers::Hawkular::DatawarehouseManager::EventCatcher
-    ManageIQ::Providers::Google::CloudManager::EventCatcher
-    ManageIQ::Providers::Kubernetes::ContainerManager::EventCatcher
-    ManageIQ::Providers::Openshift::ContainerManager::EventCatcher
-    ManageIQ::Providers::Lenovo::PhysicalInfraManager::EventCatcher
-    MiqUiWorker
-    MiqWebsocketWorker
-    MiqCockpitWsWorker
-  ).freeze
+  MONITOR_CLASS_NAMES               = MIQ_WORKER_TYPES
+  MONITOR_CLASS_NAMES_IN_KILL_ORDER = MIQ_WORKER_TYPES_IN_KILL_ORDER
```

Comment:
Looks good.  The only feedback I have is it is a bit odd to reference a global constant.  Can we put them in a small module that "thin ruby" processes can require?  Not for now, just wondering if you had reasons for making using a global constant.